[2024-07-22T02:32:11.571+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: example_dynamic_task_mapping.add_one scheduled__2022-05-24T00:00:00+00:00 map_index=1 [queued]>
[2024-07-22T02:32:11.591+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: example_dynamic_task_mapping.add_one scheduled__2022-05-24T00:00:00+00:00 map_index=1 [queued]>
[2024-07-22T02:32:11.599+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2024-07-22T02:32:11.600+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 1
[2024-07-22T02:32:11.600+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2024-07-22T02:32:11.650+0000] {taskinstance.py:1300} INFO - Executing <Mapped(_PythonDecoratedOperator): add_one> on 2022-05-24 00:00:00+00:00
[2024-07-22T02:32:11.656+0000] {standard_task_runner.py:55} INFO - Started process 1289 to run task
[2024-07-22T02:32:11.661+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'example_dynamic_task_mapping', 'add_one', 'scheduled__2022-05-24T00:00:00+00:00', '--job-id', '369', '--raw', '--subdir', '/home/***/.local/lib/python3.7/site-packages/***/example_dags/example_dynamic_task_mapping.py', '--cfg-path', '/tmp/tmpx4qne0em', '--map-index', '1']
[2024-07-22T02:32:11.663+0000] {standard_task_runner.py:83} INFO - Job 369: Subtask add_one
[2024-07-22T02:32:11.851+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 369 for task add_one ([Errno 2] No such file or directory: '/opt/airflow/logs/dag_id=example_dynamic_task_mapping/run_id=scheduled__2022-05-24T00:00:00+00:00/task_id=add_one/map_index=1/attempt=1.log'; 1289)
[2024-07-22T02:32:11.866+0000] {local_task_job.py:208} INFO - Task exited with return code 1
[2024-07-22T02:32:11.946+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
